CMP1903_A1_2324/Game.cs

```diff
@@ -16,6 +16,25 @@ internal class Game
          */
 
```

Comment:
performs the task required. The method PLay() has strange capitalisation, intentional or not